client/src/connection_cache.rs

```diff
@@ -384,114 +382,20 @@ fn get_connection(addr: &SocketAddr) -> (Connection, Arc<ConnectionCacheStats>)
     connection_cache_stats
         .get_connection_ms
         .fetch_add(get_connection_measure.as_ms(), Ordering::Relaxed);
-    (connection, connection_cache_stats)
```

Comment:
What is the point of updating the connection_cache_stats if we are not returning it?